mkdocs/docs/user_guide.md

```diff
@@ -2664,7 +2666,7 @@ If the values of `subset` is `predictions`, then only datapoints where the the m
 
 ### compare_classifiers_performance_changing_k
 
-This visualization uses the `top_k`, `ground_truth_metadata`, `field`, `probabilities` and `model_names` parameters.
+This visualization uses the `top_k`, `ground_truth`, `field`, `probabilities` and `model_names` parameters.
```

Comment:
doublecheck if this is correct

Reply:
I have checked it is correct - The new cli function prototype does not need the ground truth metadata. You can also check the old compare_classifiers_performance_changing function declaration to see that it is not using that argument.
def compare_classifiers_performance_changing_k_cli(
        probabilities,
        ground_truth,
        ground_truth_split,
        field,
        **kwargs
):